Gopkg.lock

```diff
@@ -72,7 +72,7 @@
 [[projects]]
   name = "github.com/getsentry/raven-go"
   packages = ["."]
-  revision = "563b81fc02b75d664e54da31f787c2cc2186780b"
+  revision = "master"
```

Comment:
We should use dependencies from master. When https://github.com/fabric8-services/fabric8-common/pull/20 is merged I will update dependencies to fabric8-common and raven-go